### ISSUE TITLE
Continue pre-training or fine-tune? 

### ISSUE THREAD

**Issue by bhavanadalvi** [google-research/text-to-text-transfer-transformer#384]
Hello,

I wanted to get some clarity regarding train vs fine-tune options on existing pertained T5 model. Is there a difference between the two? E.g. train will adjust weights for all layers in the model whereas fine-tune will freeze all other layers and only tune weights in the last layer?
In particular, we are interested in 
        1. First continue fine-tuning T5 on our domain corpus. It would be awesome if we have a control over the masking or kinds of queries. 
        2. Then fine-tune it further on our end task.

It is not clear how to do step-1 here. Should we use "fine-tune option for both steps" OR "use train for step-1 and fine-tune for step-2"?
If we need to use "train" option, then can you please give an example of how the dataset should be formatted and what command options should be provided? Also, how can we control the masking strategies?

**Comment by craffel** [comment]
There is no such distinction between training and fine-tuning. During fine-tuning, we update all model parameters, we just continue training (from a pre-trained model) on a new dataset. See http://tiny.cc/t5-colab for an example.

**Comment by bhavanadalvi** [comment]
Thanks Colin!
